packages/flutter/lib/src/painting/shader_warm_up.dart

```diff
@@ -72,20 +76,20 @@ abstract class ShaderWarmUp {
   /// Skia draw operations are commonly used, and which shader compilations
   /// are causing janks.
   @protected
-  void warmUpOnCanvas(ui.Canvas canvas);
+  Future<void> warmUpOnCanvas(ui.Canvas canvas);
 
   /// Construct an offscreen image of [size], and execute [warmUpOnCanvas] on a
   /// canvas associated with that image.
-  void execute() {
+  Future<void> execute() async {
```

Comment:
This has been made to return a `Future`, but nothing seems to await this future?